app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_target_parser.rb

```diff
@@ -0,0 +1,25 @@
+class ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventTargetParser
+  attr_reader :ems_event
+
+  def initialize(ems_event)
+    @ems_event = ems_event
+  end
+
+  def parse
+    parse_ems_event_targets(ems_event)
+  end
+
+  def parse_ems_event_targets(event)
+    target_collection = InventoryRefresh::TargetCollection.new(
+      :manager => event.ext_management_system,
+      :event   => event
+    )
+
+    case event[:event_type]
+    when 'is.instance.instance.create'
```

Comment:
@nasark we should probably make this more general, `when /^is\.instance/` should cover all vm events

Reply:
@agrare so we want a targeted refresh for all VM events?

Reply:
when creating an instance, that usually fires off multiple `is.instance` events and so that would mean multiple targeted refreshes would be kicked off for the same vm in a short timeframe

Reply:
Yeah that's fine, we group multiple targets into a single refresh so if they come in in rapid succession then they will be refreshed together

Reply:
for example, when an instance is created on IBM Cloud, the event service fires off `is.instance.instance.create`, `is.instance.instance.start`, and multiple `is.instance.instance.read` events in quick succession